src/ap-facebook-integration/README.md

```diff
@@ -0,0 +1,25 @@
+# Facebook Integration
+
+## Description
+
+A simple Facebook integration for creating a Case and Social Log record when a user posts something public to a facebook page
+
+## Screenshot
+
+![Facebook Integration](https://raw.githubusercontent.com/platform-experience/api-integration-library/master/src/ap-facebook-integration/images/ap-facebook-integration.png)
+
+## Additional Information/Notes
+
+You will need to have a published Facebook Page and have admin access to utilize this integration. 
+
+## Installation
+
+Download and install update set **[ap-facebook-integration.u-update-set.xml](https://github.com/platform-experience/api-integration-library/blob/master/src/ap-facebook-integration/ap-facebook-integration.u-update-set.xml)**
+
+Download and import Scheduled Job **[ap-facebook-integration-scheduled-job.xml](https://github.com/platform-experience/api-integration-library/blob/master/src/ap-facebook-integration/ap-facebook-integration-scheduled-job.xml)**
+
+* SN Product Documentation - ['Load a customization from a single XML file'](https://docs.servicenow.com/bundle/kingston-application-development/page/build/system-update-sets/task/t_SaveAnUpdateSetAsAnXMLFile.html)
```

Comment:
We can point to the NY docs now. I'll need to update the bash scripts, I know this gets autogenerated. So my fault here.